curriculum/challenges/english/08-coding-interview-prep/rosetta-code/hash-join.english.md

```diff
@@ -175,7 +175,7 @@ tests:
   - text: <code>hashJoin</code> is a function.
     testString: assert(typeof hashJoin === 'function', '<code>hashJoin</code> is a function.');
```

Comment:
```suggestion
    testString: assert(typeof hashJoin === 'function');
```

Reply:
I must've missed this one mistakenly yesterday

Reply:
This PR only removed assert message arguments which were not removed in the original two Coding Interview Prep PRs (#36410 and #36412).  If you look at those PRs, you will find this is removed on one of them.